farm/data_handler/processor.py

```diff
@@ -1671,6 +1672,168 @@ def _sample_to_features(self, sample) -> dict:
         )
         return features
 
+class DPRProcessor(Processor):
+    """
+    Used to handle the text DPR datasets that come in json format
+    """
+    def __init__(
+        self,
+        query_tokenizer,
```

Comment:
For now ok, but we need to find a proper solution to be compatible with the regular "tokenizer" arg of the base class before merging.